content/en/core/releases/3.15.0.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "3.15.0 release notes"
+linkTitle: "3.15.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+3.15.0%22).
```

Comment:
Is this a new plan going forwards to have these `Affects...` labels?

Reply:
Yes! Conversation here: https://github.com/medic/cht-core/issues/7562#issuecomment-1075733752

I'll remember to write this up permanently somewhere and communicate with team.